cmd/schemagen/testdata/models_test.go

```diff
@@ -0,0 +1,91 @@
+package schemagentest
+
+import (
+	"flag"
+	"github.com/google/go-cmp/cmp"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx/v2"
+	"github.com/scylladb/gocqlx/v2/qb"
+)
+
+var flagCluster = flag.String("cluster", "127.0.0.1", "a comma-separated list of host:port tuples")
```

Comment:
comment states "host:port" but the default is only "host"

Reply:
Fixed